config.yaml.defaults

```diff
@@ -75,17 +75,67 @@ app:
       icon: Info
       url: /about
 
-  homepage_widgets:
-    # Layouts arrays should be [x, y, w, h]
-    # See https://github.com/STRML/react-grid-layout
-
-    # component_props are properties passed down to the
-    # underlying React component, indicated by component_name
+  # See https://github.com/STRML/react-grid-layout for more on the grid package
+  homepage_grid:
+    # This section describes the grid on which Home Page widgets are laid out.
+    #
+    # The breakpoints describe screen sizes at which a different set of widget
+    # layouts should be used.
+    #
+    # The cols describe the number of evenly spaced columns that make up the
+    # grid at a given breakpoint. For example, on extra-large screens (greater
+    # than ${breakpoints.xlg} pixels), the grid will use ${cols.xlg} columns of
+    # equal width to describe sizes of widgets based on the layouts provided.
+    #
+    # Optionally, you may provide a "row_height: {number of pixels}" property in
```

Comment:
Why is the row height set to 150px?  Can we set it in rem?

Reply:
I used pixels because that's what the react-grid-layout package takes. But yeah, I guess we could take in rem and just convert it?

Reply:
We can use the same utility as MUI: https://www.npmjs.com/package/convert-css-length

See https://github.com/mui-org/material-ui/pull/14573/files#diff-c3e0d8891a5bacebd344a4f81ff36539R36 and the next 10 lines.